curriculum/challenges/_meta/lab-business-card/meta.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "Design a Business Card",
+  "isUpcomingChange": true,
+  "usesMultifileEditor": true,
+  "dashedName": "lab-business-card",
+  "order": 2,
```

Comment:
The issue about the order should be fixed, so you should be able to change this to whatever.

Reply:
Also, I updated the spreadsheet to fill in the block order - it looks like this one is 38